### PR TITLE
Adding ioxbank gateway information in help menu

### PR DESCRIPTION
**Pull request by ioBanker**

### CODE REVIEW

File: app/help/en/gateways/ioxbank.md
Comment (sschiessl-bcp) [inline app/help/en/gateways/ioxbank.md:2]: please change wording to gateway, or remove bridge. Bridge in the context of the wallet is the additional service that blocktrades offers (swap ETH to BTS).

File: app/help/en/gateways/ioxbank.md
Comment (sschiessl-bcp) [inline app/help/en/gateways/ioxbank.md:1]: Please remove the notion of "instant" as long as your service does not operate within one block.

File: app/help/en/gateways/ioxbank.md
Comment (sschiessl-bcp) [inline app/help/en/gateways/ioxbank.md:24]: Leave this to the deposit modal which generates the address on the fly from your API. If API not available, no adress. Otherwise users might use this manually and end up getting stuck, and also you might want to dynamically change this adress without having to wait for release

**Review by sschiessl-bcp — CHANGES_REQUESTED** [review]

File: app/help/en/gateways/ioxbank.md
Comment (ioBanker) [reply app/help/en/gateways/ioxbank.md:1]: Instant means once blocks are confirmed and it's the correct notion.

File: app/help/en/gateways/ioxbank.md
Comment (ioBanker) [reply app/help/en/gateways/ioxbank.md:24]: There is no address generation happening hence there is no need to use the API to show your tag; tags doesn't need to be generated, All numbers are there, they are just linked to every user id over BitShares; this XRP address is dedicated for BitShares Blockchain and wouldn't need to be changed.

Comment (ioBanker) [reply app/help/en/gateways/ioxbank.md:24]: ioxbank is offering the ability for deep integration between (BitShares Blockchain and XRP Ledger); ANY one with addresses over both the Blockchains can integrate with the instant swap service in order to swap XRP assets to BitShares assets instantly; hence at this phase of integration; users of XRP Ledger can use ioxbank to send XRP (core asset of XRP Ledger) to ANY address over BitShares blockchain and users of BitShares Blockchain can use ioxbank to send IOB.XRP (Pegged Issued Asset) to ANY XRP Ledger address.

File: app/help/en/gateways/ioxbank.md
Comment (sschiessl-bcp) [reply app/help/en/gateways/ioxbank.md:1]: I disagree that this reflects an Instant Swap, I think it is misleading.

File: app/help/en/gateways/ioxbank.md
Comment (sschiessl-bcp) [reply app/help/en/gateways/ioxbank.md:24]: The address "rDce39TBzszQJx2sshLy5jPXV3F9nEaQ9Y" is loaded from your gateway API when requesting deposit, is it not? I have not seen this hardcoded in the UI deposit modal code?

Comment (ioBanker) [reply app/help/en/gateways/ioxbank.md:24]: Yes it's not hardcoded within the UI but it's a static address over the API and it will not be changed.

Please refer to address info: https://livenet.xrpl.org/accounts/rDce39TBzszQJx2sshLy5jPXV3F9nEaQ9Y as this address belongs to the domain ioxbank.com

File: app/help/en/gateways/ioxbank.md
Comment (ioBanker) [reply app/help/en/gateways/ioxbank.md:1]: Perhaps together "Instant Swap" would be misleading to "Atomic Swap"; I agree with this.

I can change to "Instant Gateway Service", do you agree with this? I still believe "instant" is relevant as the transaction happens as instant as possible and time is only required for proofing the Blocks confirmation which take seconds.

File: app/help/en/gateways/ioxbank.md
Comment (sschiessl-bcp) [reply app/help/en/gateways/ioxbank.md:24]: My point is: If you ever need to change the adress for whatever reason, or disable deposits you can do that. If you hardcode the address anywhere, you can't, or shouldn't.

In any case, I guess this is within the responsibility of you as the operator, so if you want to keep it that way, it's fine. I do recommend against hardcoding though

File: app/help/en/gateways/ioxbank.md
Comment (sschiessl-bcp) [reply app/help/en/gateways/ioxbank.md]: Instant Gateway Service is acceptable, please do clarify in the text what instant means.

File: app/help/en/gateways/ioxbank.md
Comment (ioBanker) [reply app/help/en/gateways/ioxbank.md:24]: I will keep it this way; thanks.

File: app/help/en/gateways/ioxbank.md
Comment (ioBanker) [reply app/help/en/gateways/ioxbank.md]: > Instant Gateway Service is acceptable to me, please do clarify in the text what instant means.

done: https://github.com/bitshares/bitshares-ui/blob/f63df4bc759a408520c4a3e85fcad8a2934e8cad/app/help/en/gateways/ioxbank.md

**Pull request merged**
